models/gamesM.js

```diff
@@ -178,3 +179,13 @@ exports.selectUsers = () => {
     return users.rows;
   });
 };
+
+exports.removeComment = (comment_id) => {
+  return checkCommentExists(comment_id).then(() => {
+    return pool
+      .query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id])
+      .then((comment) => {
+        return comment.rows[0];
```

Comment:
You do not need to return here, but what is comment.rows[0]? could this be used to check if this comment exists?